### ISSUE TITLE
[REGRESSION]: Can no longer click Material UI select box

### ISSUE THREAD

**Issue by alisterscott** [microsoft/playwright#10627]
**Context:**
- GOOD Playwright Version: 1.16.2
- BAD Playwright Version: 1.17.0
- Operating System: Mac and Linux

**Code Snippet**

```
  test('can work with material UI select list', async ({ page }) => {
    await page.goto('https://mui.com/components/selects/')
    await page.click('#demo-simple-select')
    await page.click('li[data-value="30"]')
  })
```

shows errors now:

```
Timeout of 30000ms exceeded.
Pending operations:
  - page.click at scenarios/alltests.spec.ts:79:16

page.click: Target closed
=========================== logs ===========================
waiting for selector "#demo-simple-select"
  selector resolved to visible <div tabindex="0" role="button" aria-expanded="false"…>…</div>
attempting click action
  waiting for element to be visible, enabled and stable
  element is visible, enabled and stable
  scrolling into view if needed
  done scrolling
  performing click action
  <li tabindex="-1" role="option" data-value="30" aria…>…</li> from <div id="menu-" role="presentation" class="MuiModal-r…>…</div> subtree intercepts pointer events
```

**Describe the bug**

This was working in playwright 1.16.x 

Also, for some reason the HTML report locally doesn't show images or trace any more

**Comment by denny99** [comment]
There is definitely a new behaviour in playwright 1.17.0.

In my case, i have a toast message above a button.
And when executing the click playwright does the following

```
    retrying click action, attempt #57
      waiting 500ms
      waiting for element to be visible, enabled and stable
      element is visible, enabled and stable
      scrolling into view if needed
      done scrolling
      performing click action
```

The problem is, playwright thinks that the element is visible and moves the mouse to the button. Our toasts are automatically closed BUT when hovering them with the mouse, this mechanism is paused until the mouse leaves the toast. So the toast never disappears und our test fails.

When reverting back to 1.16.3 it is working again.

**Comment by alisterscott** [comment]
I think this has since been reported here: https://github.com/microsoft/playwright/issues/10632

**Comment by dgozman** [comment]
@alisterscott Could you please share a trace file with us? You can send it to dgozman [at] microsoft [dot] com if you'd like. Here are instructions to record a trace: https://playwright.dev/docs/trace-viewer#recording-a-trace.

**Comment by pavelfeldman** [comment]
@alisterscott: could you confirm that running the test with the following environment variable PLAYWRIGHT_NO_LAYOUT_SHIFT_CHECK=1 resolves the issue?

**Comment by alisterscott** [comment]
@pavelfeldman 

> could you confirm that running the test with the following environment variable PLAYWRIGHT_NO_LAYOUT_SHIFT_CHECK=1 resolves the issue?

yes this works

**Comment by alisterscott** [comment]
@dgozman 
 
> Could you please share a trace file with us? 

Emailed just now

**Comment by dgozman** [comment]
Thank you, this will be fixed in v1.17.1 by #10653.
